deploy/roles/postgresql-server-config/tasks/main.yml

```diff
@@ -157,9 +169,10 @@
   check_mode: no
   register: result
   failed_when:
-    - 'result.module_stdout is defined and "extension \"citus\" does not exist" not in result.module_stdout'
-    - 'result.module_stderr is defined and "extension \"citus\" does not exist" not in result.module_stderr'
+    - '"extension \"citus\" does not exist" in result.notices'
```

Comment:
We can probably simplify the quotes to: `'extension "citus" does not exist' in result.notices` (omitting outermost quotes)

Reply:
The problem is that if a statement starts with quotes, Ansible expects the whole thing to be a string to pass internally to `eval`, so it fails unless the double quoting is used.